CONTRIBUTING.md

```diff
@@ -62,26 +70,45 @@ Tests also depend on NeTEx specification that are imported as a git submodule.
 Therefore, these tests are run only if feature `xmllint` is activated.\
 
 To install xmllint and submodules:
+
 ```sh
 git submodule update --init --recursive
 apt install libxml2-utils
 ```
 
-#### Launch all tests
+#### Check outputs manually
+
+To validate the NeTEx output it is possible to use xmllint:
 
 ```sh
-# Run all the tests of `transit_model` in the entire repository,
-# activating all features, including `xmllint`
-cargo test --workspace --all-features
+xmllint --noout --nonet --huge --schema /path/to/NeTEx/xsd/NeTEx_publication.xsd your_file.xml
 ```
 
-#### Hand-check outputs
+Note: this may be very (very) slow on huge files.
+
+#### Launch all tests
 
-To validate the output NeTEx obtained it is possible to use xmllint:
 ```sh
-xmllint --noout --nonet --huge --schema /path/to/NeTEx/xsd/NeTEx_publication.xsd your_file.xml
+# Run all the tests of `transit_model` in the entire repository,
+# without fixtures, then activating all features, including `xmllint`
```

Comment:
what does it mean "without fixtures"?
`cargo test --workspace` does not run test with fixtures?

Reply:
I meant "features". Changed!